src/DataContainer/Page.php

```diff
@@ -5,53 +5,37 @@
 namespace Terminal42\FolderpageBundle\DataContainer;
 
 use Contao\Backend;
-use Contao\Config;
 use Contao\CoreBundle\Exception\RedirectResponseException;
 use Contao\Database;
 use Contao\DataContainer;
+use Contao\Image;
 use Contao\Input;
+use Contao\Message;
 use Contao\PageModel;
+use Contao\StringUtil;
+use Contao\System;
 use Doctrine\DBAL\Connection;
 use Symfony\Component\HttpFoundation\RequestStack;
-use Symfony\Component\HttpFoundation\Session\Attribute\AttributeBagInterface;
+use Symfony\Component\HttpFoundation\Session\SessionBagInterface;
 use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Symfony\Component\Routing\RouterInterface;
 use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
 
 class Page
 {
-    /**
-     * @var Connection
-     */
-    private $db;
+    private Connection $db;
 
-    /**
-     * @var RequestStack
-     */
-    private $requestStack;
+    private RequestStack $requestStack;
 
-    /**
-     * @var AttributeBagInterface
-     */
-    private $session;
+    private SessionBagInterface $session;
 
-    /**
-     * @var RouterInterface
-     */
-    private $router;
+    private RouterInterface $router;
 
-    /**
-     * @var \BackendUser
-     */
-    private $user;
-
-    public function __construct(
-        Connection $db,
-        RequestStack $requestStack,
-        SessionInterface $session,
-        RouterInterface $router,
-        TokenStorageInterface $tokenStorage
-    ) {
+    private UserInterface $user;
```

Comment:
I'm not sure if it is also `UserInterface` in Contao 4.9 - but `\BackendUser` was not compatible with `$tokenStorage->getToken()->getUser();`